.tests/csf-lfd-logs/config.yaml

```diff
@@ -1,8 +1,8 @@
 parsers:
 - crowdsecurity/syslog-logs
-- crowdsecurity/configserver-lfd-logs.yaml
+- crowdsecurity/configserver-lfd-logs
```

Comment:
i think it should be `crowdsecurity/s01-parse/configserver-lfd-logs.yaml`

Reply:
sorry i had renammed csf-lfd-logs to configserver-lfd-logs also, so it also gonna mess the asserts

Reply:
maybe

Reply:
yes I figured, thanks